docs/source/creating_an_image_processing_pipeline/index.rst

```diff
@@ -142,6 +148,88 @@ Finding and Decoding Spots
+
+There are many approaches for segmenting cells from image-based transcriptomics assays. Below are
+a few methods that are implemented or integrated with starfish to output a
+:py:class:`.BinaryMaskCollection`, which represents a collection of labeled objects. If you do not
+know which segmentation method to use, a safe bet is to start with thresholding and watershed. On
+the other hand, if you can afford to manually define ROI masks there is no better way to
+guarantee accurate segmentation.
+
+.. note::
+    While there is no "ground truth" for cell segmentation the closest approximation is manual
+    segmentation by an expert in the tissue of interest.
+
+Thresholding and Watershed
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The traditional method for segmenting cells in fluorescence microscopy images is to threshold the
+image into foreground pixels and background pixels and then label connected foreground as
+individual cells. Common issues that affect thresholding such as background noise can be corrected
+by preprocessing images before thresholding and filtering connected components after. There are
+`many automated image thresholding algorithms <https://imagej.net/Thresholding>`_ but currently
+starfish requires manually selecting a global threshold value in :py:class:`.ThresholdBinarize`.
+
+When overlapping cells are labeled as one connected component, they are typically segmented by
+using a `distance transformation followed by the watershed algorithm <https://www.mathworks
+.com/company/newsletters/articles/the-watershed-transform-strategies-for-image-segmentation
+.html>`_. Watershed is a classic image processing algorithm for separating objects in images and
+can be applied to all types of images. Pairing it with a distance transform is particularly
+useful for segmenting convex shapes like cells.
+
+A segmentation pipeline that consists of thresholding, connected component analysis, and watershed
+is the simplest and fastest to implement but its accuracy is highly dependent on image quality.
+The signal-to-noise ratio of the cell stain must be high enough for minimal errors after
+thresholding and binary operations. And the nuclei or cell shapes must be convex to meet the
+assumptions of the distance transform or else it will over-segment. Starfish includes the basic
+functions to build a watershed segmentation pipeline and a predefined :py:class:`.Watershed`
+segmentation class that uses the :term:`primary images<Primary Images>` as the cell stain.
+
+* Tutorial: :ref:`Ways to segment by thresholding and watershed<tutorial_watershed_segmentation>`
+
+Manually Defining Cells
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The most accurate but time-consuming approach is to manually segment images using a tool such as
+`ROI manager <https://imagej.net/docs/guide/146-30.html#fig:The-ROI-Manager>`_ in FIJI (ImageJ). It
+is a straightforward process that starfish supports by allowing ROI set binaries to be imported as a
```

Comment:
"ROI set binaries" is confusing to me.  Do you mean ROI sets saved as binary files?

Reply:
Yes that's what I meant... I got the language from some ROI manager documentation I can't find now but I agree it's confusing. How about "that starfish supports by importing ROI sets stored in ZIP archives".

Reply:
sounds good.